{{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/plugin.py

```diff
@@ -45,35 +49,78 @@
 # INITIALIZATION TASKS
 ########################################
 
-# To run the script from templates/{{ cookiecutter.plugin_name }}/tasks/myservice/init, add:
-# hooks.Filters.COMMANDS_INIT.add_item((
-#     "myservice",
-#     ("{{ cookiecutter.plugin_name }}", "tasks", "myservice", "init"),
-# ))
+# To add a custom initialization task, create a bash script template under:
+#   {{ cookiecutter.module_name }}/templates/{{ cookiecutter.plugin_name }}/jobs/init/
+# and then add it to the MY_INIT_TASKS list. Each task is in the format:
+#   ("<service>", ("<path>", "<to>", "<script>", "<template>"))
+MY_INIT_TASKS: list[tuple[str, tuple[str, ...]]] = [
+    ("lms", ("{{ cookiecutter.plugin_name }}", "jobs", "init", "lms.sh")),
+    ("cms", ("{{ cookiecutter.plugin_name }}", "jobs", "init", "cms.sh")),
+    ("mysql", ("{{ cookiecutter.plugin_name }}", "jobs", "init", "mysql.sh")),
+]
+
+
+# For each task added to MY_INIT_TASKS, we load the task template
+# and add it to the CLI_DO_INIT_TASKS filter, which tells Tutor to
+# run it as part of the `init` job.
+for service, template_path in MY_INIT_TASKS:
+    full_path = pkg_resources.resource_filename(
+        "{{ cookiecutter.module_name }}", os.path.join("templates", *template_path)
+    )
+    with open(full_path, encoding="utf-8") as init_task_file:
+        init_task = shlex.join(["bash", "-c", init_task_file.read()])
```

Comment:
There is no need to wrap the scripts in `bash -c "..."`.  Scripts are already wrapped in `sh -e -c "..."`: https://github.com/overhangio/tutor/blob/nightly/tutor/commands/compose.py#L388
Also, bash may not be available in the container. (only sh)

Reply:
👍🏻